docs/docs/recipes/working-with-themes.md

```diff
@@ -4,6 +4,72 @@ title: "Recipes: Working with Themes"
 
 A [Gatsby theme](/docs/themes/what-are-gatsby-themes) abstracts Gatsby configuration (shared functionality, data sourcing, design) into an installable package. This means that the configuration and functionality isn’t directly written into your project, but rather versioned, centrally managed, and installed as a dependency. You can seamlessly update a theme, compose themes together, and even swap out one compatible theme for another.
 
+## Creating a new site using a theme
+
+Found a theme you'd like to use on your project? Awesome! You can work through setting up by following the steps below.
+
+> If you'd like to take a look at more theme options, you could take a look at this [list of themes](https://www.npmjs.com/search?q=gatsby-theme) .
+
+### Prerequisities
+
+The [Gatsby CLI](/docs/gatsby-cli) installed
+
+### Directions
+
+1. Create a gatsby site
+
+```shell
+gatsby new {your-project-name}
+```
+
+2. Change directory and install theme
+
+```shell
+cd {your-project-name}
+npm install {theme-name}
+```
+
+For Example
+
+```shell
+npm install gatsby-theme-blog
+
+```
+
+3. Add theme to `gatsby.config.js`
+
+```shell
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-theme-blog`,
+      options: {
+        /*
+        - basePath defaults to `/`
+        - contentPath defaults to `content/posts`
+        - assetPath defaults to `content/assets`
+        - mdx defaults to `true`
+        */
+        basePath: `/blog`,
+        contentPath: `content/blogPosts`,
+        assetPath: `content/blogAssets`,
+        mdx: false,
+      },
+    },
+  ],
+}
+```
+
+4. Run `yarn develop` , the theme should be avaliable at `http://localhost:8000/{basePath}`
```

Comment:
Based on this we likely need to include yarn installation as a pre-req as well? Or can they run gatsby develop instead?

Reply:
I'll add yarn to the pre-req. It's used mostly with themes because of Yarn workspaces.

Reply:
For sure, my question is whether workspace are necessary for using themes? Or just developing them? The [theme starter example](https://www.gatsbyjs.org/docs/recipes/working-with-themes#directions-1) uses `gatsby develop` so figured it was worth double-checking. I don't actually know the answer, so if you're up to test it out that'd be awesome.

Reply:
I don't think workspaces are necessary for using themes. Also, I saw that running yarn develop triggers gatsby develop.

Reply:
So let's try using just `gatsby develop` and if it's successful you can open a new PR making the change and removing the yarn dependency :)

Reply:
Hmm, I'd love to understand this a bit better. You're saying that when you create a new gatsby project it's creating a yarn.lock file? That shouldn't be the case as `gatsby new` uses npm. Would love to see what steps you took and where the yarn.lock came into play! I'm not inclined to include any notes about yarn if you don't need to use it in this scenario. I'm wondering if the issues you're seeing are because you already had?

Reply:
if someone uses before `gatsby` in combination with `yarn`, then `yarn` is used instead of `npm`.

Reply:
Thanks for pointing this out @muescha we fixed it in #20965